src/Packetery/Module/Options/Repository.php

```diff
@@ -67,4 +58,34 @@ public function deleteExpiredTransientsByPrefix( string $prefix ) {
 			)
 		);
 	}
+
+	/**
+	 * Delete transients by prefix.
+	 *
+	 * @param string $prefix Custom transient prefix.
+	 *
+	 * @return int|bool
+	 */
+	public function deleteTransientsByPrefix( string $prefix ) {
+		$transientPrefix = sprintf( '_transient_%s', $prefix );
+
+		return $this->deleteByPrefix( $transientPrefix );
```

Comment:
V databázi zůstanou timeout záznamy ke smazaným transientům. Určitě by měla být použita funkce delete_transient.

Reply:
Jestli neexistuje vhodná funkce WP, tak tahat z db v repozitáři a v rámci jiné třídy mazat dedikovanou funkcí. Stejně tak pro options. Analýza i návrh řešení jsou v tvých rukou.